repos/spack_repo/builtin/packages/libproxy/package.py

```diff
@@ -23,32 +23,18 @@ class Libproxy(CMakePackage):
     version("0.4.13", sha256="d610bc0ef81a18ba418d759c5f4f87bf7102229a9153fb397d7d490987330ffd")
 
     variant("perl", default=False, description="Enable Perl bindings")
-    variant("python", default=False, description="Enable Python bindings", when="@0.4.16:")
```

Comment:
Could you change this to
```
    variant("python", default=False, description="Enable Python bindings", when="@0.4.18:")
```
and revert the rest of the changes in this package besides removing the `extends` directive?

That seems a nice middleground that you don't have to add to the burden of this PR by updating libproxy but we don't make our lives more difficult when we do add newer versions.